pages/samvera/developer_resources/customize_metadata/hyrax_1.0/model.md

```diff
@@ -31,20 +31,19 @@ end
 
 ### Basic metadata
 
-Basic metadata properties are defined in [app/models/concerns/hyrax/basic_metadata.rb](https://github.com/samvera/hyrax/blob/1-0-stable/app/models/concerns/hyrax/basic_metadata.rb)
```

Comment:
I think the plan here is to support versioning of the docs (1.0, 2.0, etc.) because not all users will be following the master branch. Therefore, I think we should leave this page alone for this PR. Please remove it from the PR. I'm ready to accept all the screenshots and changes to the Manager Guide.